vs_utils/utils/PDBTransformer.py

```diff
@@ -0,0 +1,259 @@
+#Written by Evan N. Feinberg at Stanford University, contact: enf@stanford.edu
+
+
+from nnscore_pdb import PDB 
+from copy import deepcopy
+import numpy as np
+import pickle 
+
+'''Pseudocode 
+
+1. Create objects of class PDB for protein and ligand
+2. Compute centroid of ligand
+3. translate coordinates of all other atoms such that ligand centroid is now origin, for both 
+  ligand PDB and protein PDB
+4. combine PDB objects
+5. Write out new PDB (.pdb and pickle of PDB() object)
+'''
+
+'''
+Pseudocode, detail: 
+
+1. Create objects of class PDB for protein and ligand 
+2. Compute centroid of ligand
+  a. define a function for computing centroid given a PDB object. Maybe extend to make it 
+    accept just coordinates?
+    -further decomposition:
+      Take object as input --> convert to coordinates --> then send to separate method 
+      that computes centroid 
+3. For each atom for each PDB oject: 
+  -change x, y, and z coordinates by subtracting out centroid x, y, and z
+4. Function that combines PDB objects: 
+  def mergePDBs(pdb1, pdb2)
+    copy pdb1
+    add all from pdb2 to pdb1, but offset atom indices by max(atom index of pdb 1)
+'''
+
+
+def compute_centroid(molecule):
+  '''given molecule, an instance of class PDB, compute the x,y,z centroid of that molecule'''
+
+  coordinates = [molecule.all_atoms[atom].coordinates.coords for atom in molecule.all_atoms]
+  centroid = np.sum(coordinates, axis=0)/float(len(coordinates))
+  return(centroid)
+
+def generate_random_unit_vector():
```

Comment:
PDBTransformer will likely be deprecated completely. Should we go ahead with that and stick with the new architecture, i.e. grid_featurizer being all-encompassing?